back/lib/participation_method/base.rb

```diff
@@ -66,8 +66,8 @@ def posting_allowed?
       false
     end
 
-    def never_update?
-      false
+    def update_if_published?
+      true
```

Comment:
This method changes as now you can update surveys when in draft - just not when published